src/utils/py/daos_api.py

```diff
@@ -34,6 +34,15 @@
 from daos_cref import *
 from conversion import *
 
+# pylint: disable=no-member
+# pylint: disable=exec-used
+# pylint: disable=import-error
```

Comment:
Do you intend to disable these for this whole file or just a subset of it?  IIRC, `pylint: disable` disables a check until you `pylint: enable` it again.

Reply:
Will just keep "import-error", enable it back after the "import" code.